ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/monitor-session-default-encapsulation-gre.cfg

```diff
@@ -3,5 +3,3 @@ interface Management1
    description oob_management
    vrf MGMT
    ip address 10.73.255.122/24
-!
```

Comment:
This config is removed? Maybe a typo somewhere.

Reply:
Missed it.. fixed!